index.js

```diff
@@ -35,6 +35,14 @@ module.exports.pitch = function (request) {
 		"}"
 	].join("\n");
 
+        var insertInto = "void 0";
```

Comment:
Why `void 0` ?

Reply:
In this place https://github.com/savelichalex/style-loader/blob/fe58dbe66444852d9133a00d2d1883eb9036ff41/index.js#L55 I need some explicit value. This can be `undefined` or `null` of course)